internal/proxy/proxy.go

```diff
@@ -23,6 +24,18 @@ const (
 	DefaultBufferSizeBytes = 32 * 1024
 	// DefaultFlushInterval TODO
 	DefaultFlushInterval = time.Duration(-1)
+	// DefaultTimeout TODO
+	DefaultTimeout = 30 * time.Second
```

Comment:
It seems that most of these consts exposed as public identifiers are never used outside of this package. Can we make them unexported if they are not used?